packages/shared/user-data/GraphQL/config.ts

```diff
@@ -43,6 +43,12 @@ export const config = {
     label: "Disable using incremental snapshots",
     legacyKey: "devtools.features.disableIncrementalSnapshots",
   },
+  backend_enableHasAnnotationKindQueryStorage: {
```

Comment:
Nit: Can we keep these keys alpha-sorted? Helps reduce likelihood of merge conflicts and makes it easier to scan code for a specific key.